tutorials/plot_visualize_evoked.py

```diff
@@ -23,20 +23,21 @@
 ###############################################################################
 # Notice that ``evoked`` is a list of :class:`evoked <mne.Evoked>` instances.
 # You can read only one of the categories by passing the argument ``condition``
-# to :func:`mne.read_evokeds`. To make things more simple for this tutorial, we
-# read each instance to a variable.
-evoked_l_aud = evoked[0]
-evoked_r_aud = evoked[1]
-evoked_l_vis = evoked[2]
-evoked_r_vis = evoked[3]
+# to :func:`mne.read_evokeds`.
+# The read evoked objects can be stored in a dict for easier access.
+conditions = ["Left Auditory", "Right Auditory", "Left visual", "Right visual"]
+evoked = dict()
+for condition in conditions:
+    evoked[condition] = mne.read_evokeds(fname, baseline=(None, 0),
+                                          proj=True, condition=condition)
```

Comment:
@jona-sassenhagen think about new mne users with limited python skills. This is one of the first tutorial they will read. I tried to keep it simple (no zip, enumerate) with little python kung fu...

I would prefer to keep this tutorial simple and stupid. Makes sense?